README.md

```diff
@@ -1 +1,105 @@
 # mender-client-python-example
+
+Mender is an open source over-the-air (OTA) software updater for embedded Linux
+devices. Mender comprises a client running at the embedded device, as well as a
+server that manages deployments across many devices.
+
+![Mender
+logo](https://raw.githubusercontent.com/mendersoftware/mender/master/mender_logo.png)
+
+This repository contains simple example applications in Python that communicate
+with the Mender client on a device via D-Bus to demonstrate some of the use
+cases.
+
+The code in this repository is meant to educate and showcase the Mender D-Bus
+API capabilities. Please keep in mind that for simplicity error handling and
+failure recovery is not implemented in most cases, and we strongly discourage
+the use of the code as-is in production applications.
+
+## Documentation
+
+### API documentation
+
+The D-Bus APIs of the Mender client are documented in the [Device-side API
+section of the Mender Docs](https://docs.mender.io/device-side-api).
+
+### D-Bus documentation
+
+A basic understanding of the D-Bus concepts are required in order to understand
+the code in this repository. You can find an introduction [introduction to D-Bus
+here](https://www.freedesktop.org/wiki/IntroductionToDBus/).
+
+### Python PyD-Bus package
```

Comment:
Maybe one search-n-replace too much? We should not replace official names, should we? :-)

Reply:
Yes, indeed. Thanks!